src/ServiceControl.Monitoring.Data/ServiceControl.Monitoring.Data.csproj

```diff
@@ -1,9 +1,17 @@
 <Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
   </PropertyGroup>
+
   <PropertyGroup>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <Description>A ServiceControl.Monitoring wire format protocol serializer and a reporter.</Description>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <PackageProjectUrl>https://particular.net/</PackageProjectUrl>
```

Comment:
Isn't this supposed to include the package name?

Reply:
Being given that https://github.com/Particular/ServiceControl.Monitoring.Data/blob/ada4931bac4eee6c5f6f2fc665c24cbab8437a86/src/Directory.Build.targets#L11 already contains a link to the package, I'll remove it and check the output build result.

Reply:
This is not a public project and is shipped only as source, internally, to our MyGet. It's not important.